charts/retool/templates/_helpers.tpl

```diff
@@ -23,6 +23,30 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Set service names
+*/}}
+{{- define "retool.codeExecutor.name" -}}
+  {{- default (printf "%s-code-executor" (include "retool.fullname" .)) .Values.codeExecutor.name -}}
+{{- end -}}
```

Comment:
For these long nested calls it might be nicer to start doing some lispy formatting, like this. I lose track of parens after like 2 levels.
```suggestion
{{- define "retool.codeExecutor.name" -}}
  {{- default 
        (printf "%s-code-executor" (include "retool.fullname" .)) 
        .Values.codeExecutor.name 
  -}}
{{- end -}}
```

Reply:
oh whoops, for some reason github UI said this was still blocked on review.

Reply:
will address in subsequent pr!